uiowa/uiowa
# UI Press book content layout and design

Resolves #4657 

## How to test ##
Pull down branch
`blt ds --site=uipress.uiowa.edu`
`drush @uipress.local uli`
Trying adding a book content type. Use a book from the current site to make it easier/allow for proper testing of cart options.
Test the Add to Cart and View Cart buttons. Make sure the formats added (ebook, paperback, etc) show up as options to add to cart. 
Test that the sales price and sale code display when within a sale date.
